mesheryctl/pkg/utils/helpers.go

```diff
@@ -585,9 +585,9 @@ func ValidId(args string, configuration string) (string, bool, error) {
 }
 
 // ValidId - Check if args is a valid name or a valid name prefix and returns the full name and ID
-func ValidName(args string, configuration string) (string, string, bool, error) {
+func ValidName(baseUrl, args string, configuration string) (string, string, bool, error) {
```

Comment:
Can rename `baseUrl` to `mesheryServerUrl`

Reply:
Please resolve all open comments that have been addressed.